third_party/java/arcs/build_defs/internal/kotlin.bzl

```diff
@@ -58,6 +59,34 @@ def arcs_kt_binary(name, srcs = [], deps = [], visibility = None):
         kt_target = ":" + name,
     )
 
+def kt_jvm_and_wasm_library(
+        name = None,
+        srcs = [],
+        deps = [],
+        visibility = None,
+        **kwargs):
+    """Simultaneously defines JVM and WASM kotlin libraries.
+    name: String; Name of the library
+    srcs: List; List of sources
+    deps: List; List of dependencies
+    visibility: List; List of visibilities
```

Comment:
I think you'll need to format this properly, otherwise buildifier will get upset:

```
    """Simultaneously defines JVM and WASM kotlin libraries.

    Args:
      name: String; Name of the library
      srcs: List; List of sources
      deps: List; List of dependencies
      visibility: List; List of visibilities
    """
```